app/controllers/declaration/SealController.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.declaration
+
+import config.AppConfig
+import controllers.actions.{AuthAction, JourneyAction}
+import controllers.declaration.routes.SummaryPageController
+import controllers.util.CacheIdGenerator.cacheId
+import controllers.util.MultipleItemsHelper.{add, remove, saveAndContinue}
+import controllers.util.{Add, FormAction, Remove, SaveAndContinue}
+import forms.declaration.Seal
+import forms.declaration.Seal._
+import handlers.ErrorHandler
+import javax.inject.Inject
+import models.requests.JourneyRequest
+import play.api.data.Form
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, Result}
+import services.CustomsCacheService
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.declaration.seal
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class SealController @Inject()(
+  override val messagesApi: MessagesApi,
+  authenticate: AuthAction,
+  journeyType: JourneyAction,
+  errorHandler: ErrorHandler,
+  cacheService: CustomsCacheService
+)(implicit ec: ExecutionContext, appConfig: AppConfig)
+    extends FrontendController with I18nSupport {
+
+  def form(): Form[Seal] = Form(formMapping)
+  val sealsAllowed = 9999
```

Comment:
The same here, I'll move it to Seal object

Reply:
done